CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 1.1.0 - 2020-12-11
+
+### Added
+- Support debugging .NET 5 projects
+- Modified post-debug behavior so that "func host start" output remains viewable and to optimistically fix issue where VS Code becomes unresponsive [#1401](https://github.com/microsoft/vscode-azurefunctions/issues/1401)
+- Added "Reload Templates" option when creating a function that will also clear the template cache (Use setting "azureFunctions.showReloadTemplates" to enable)
```

Comment:
Added Added 😏

Reply:
You mean the header and first word in the bullet? I do this all the time haha